cpp/src/parquet/statistics.cc

```diff
@@ -620,7 +626,11 @@ class TypedStatisticsImpl : public TypedStatistics<DType> {
  private:
   const ColumnDescriptor* descr_;
   bool has_min_max_ = false;
+  // Currently, has_null_count_ is always true, and would
```

Comment:
This is confusing because the default value below is `false`

Reply:
Yes, default value is false, but ctor will ensure that it is true. I can update it to `true`.